CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Change log
 ==========
 
+0.19.6 (2021-05-12)
+-------------------
+
+New features
+~~~~~~~~~~~~
+
+* **analyzers:** add Mypy checks and inline commenting support
```

Comment:
I would say it gives users the possibility to report static type checking issues for python as inline comments.

Not every user knows what is mypy.